tools/ml-dataprep/run.sh

```diff
@@ -0,0 +1,8 @@
+python runner.py \
```

Comment:
Copyright line in script?

Reply:
Fixed in dc4430f316956c837b2ae237e4ea85a9489cb8fe